pwndbg/disasm/__init__.py

```diff
@@ -85,7 +85,7 @@ def get_disassembler(pc):
     if pwndbg.arch.current in ('arm', 'aarch64'):
         extra = {0:CS_MODE_ARM,
                  0x20:CS_MODE_THUMB}[pwndbg.regs.cpsr & 0x20]
-
+    pwndbg.arch.mode = pwndbg.arch.MODES[extra]
```

Comment:
The `get_disassembler_cached` seems to be called just from `get_disassembler`. And here it would be either `None`, `CS_MODE_ARM` or `CS_MODE_THUMB`.

I guess currently it can't be out of `pwndbg.arch.MODES`?
We can change it to `.get(extra, None)` so that future changes won't break stuff, but I guess we can leave it as it is?